lib/paperclip/media_type_spoof_detector.rb

```diff
@@ -62,19 +62,21 @@ def media_types_from_name
       @media_types_from_name ||= content_types_from_name.collect(&:media_type)
     end
 
-    def calculated_content_type
-      @calculated_content_type ||= type_from_file_command.chomp
+    def calculated_content_types
+      @calculated_content_types ||= types_from_file_command.collect{|t| t.chomp}.uniq
```

Comment:
Line is too long. [85/80]<br>Prefer `map` over `collect`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:chomp` as an argument to `collect` instead of a block.